infrastructure/removePublicGCS/README.md

```diff
@@ -0,0 +1,34 @@
+# removePublicGCS - Sample 5-minute Solution to prevent public objects in GCS via Cloud Functions
+
+This is a sample of deploying a *Cloud Function* that triggers off a target *GCS* bucket whenever an object's metadata or ACLs are updated. For each object that is updated, this function will explicitly remove the **'allAuthenticatedUsers'** and **'allUsers'** special ACL groups from the owners, readers, and writers groups. This function effectively reacts and removes public access if it is ever accidentally granted by a user or code. The object getting the public ACLs removed are not modified in any other way, the object data, metadata, and other ACLs will remain the same after the function triggers.
+
+## Steps
+
+Begin by activating a *Cloud Shell* within your [Cloud Console](https://console.cloud.google.com/).
+
+    ## Make a new directory as a staging area to deploy the cloud function
+    $ mkdir removePublicAccessGCF
+    $ cd removePublicAccessGCF
+
+
+    ## Grab a copy of the sample Cloud Function code
+    $ wget https://raw.githubusercontent.com/reechar-goog/professional-services/feature/removePublicGCS_CF/infrastructure/removePublicGCS/index.js
+    $ wget https://raw.githubusercontent.com/reechar-goog/professional-services/feature/removePublicGCS_CF/infrastructure/removePublicGCS/package.json
+
+
+    ## Use NPM to install a copy of the dependencies 
+    $ npm install --save @google-cloud/storage
```

Comment:
Switch the command to rely on your package.json to list the dependencies instead.